core/src/test/java/org/infinispan/partitionhandling/BaseTxPartitionAndMergeTest.java

```diff
@@ -358,5 +362,12 @@ public void await(long timeout, TimeUnit timeUnit) throws InterruptedException {
       public void unblock() {
          collection.forEach(BaseTxPartitionAndMergeTest.AwaitAndUnblock::unblock);
       }
+
+      public void disableDiscard() {
+         collection.stream()
+               .map(awaitAndUnblock -> awaitAndUnblock instanceof DiscardFilter ? (DiscardFilter) awaitAndUnblock : null)
```

Comment:
`.filter(DiscardFilter.class::isInstance).map(DiscardFilter.class::cast)`

Reply:
:+1:

Reply:
I changed the name to `stopDiscard()`